frontend/components/MostViewed.tsx

```diff
@@ -1,44 +1,149 @@
 import React, { Component } from 'react';
 import { css } from 'react-emotion';
-import { headline, textEgyptian } from '@guardian/pasteup/fonts';
+import { headline, egyptian } from '@guardian/pasteup/fonts';
 import { palette } from '@guardian/pasteup/palette';
-import { desktop } from '@guardian/pasteup/breakpoints';
+import {
+    desktop,
+    mobileLandscape,
+    tablet,
+    leftCol,
+} from '@guardian/pasteup/breakpoints';
 import { BigNumber } from '@guardian/guui';
 
+const container = css`
+    border-top: 1px solid #dcdcdc;
+    padding-top: 3px;
+
+    ${desktop} {
+        padding-top: 6px;
+    }
+`;
+
 const heading = css`
     font-family: ${headline};
     color: ${palette.neutral[7]};
-    font-size: 20px;
-    line-height: 1.2;
+    font-size: 24px;
     font-weight: 900;
-    margin-top: 27px;
-    margin-bottom: 1px;
+    line-height: 1;
+    padding-right: 5px;
+    padding-bottom: 4px;
+
+    ${leftCol} {
+        font-size: 20px;
+        line-height: 1.2;
+        width: 140px;
+        position: relative;
+
+        :after {
+            content: '';
+            display: block;
+            position: absolute;
+            height: 30px;
+            width: 1px;
+            background-color: #dcdcdc;
```

Comment:
Those inline colour hexes... @aware's inline hex regex be comin atcha

Reply:
😨 uh oh! Yep I'll replace those with palette references right away!